### PR TITLE
Profile page 

### PR DESCRIPTION
**Pull request by narharikale**
added useSlice
added profile page
added profile edit modal

### CODE REVIEW

**Review by MiheerTamkhane — APPROVED** [review]
Everything is good like the code structure, DB section, and post section.
The UI is also smooth just add the env var and netlify.toml file 
and you good to go.
Nice work

**Review by rahulmistry751 — APPROVED** [review]
Clean UI and Code 
Good to merge

File: src/frontend/components/post/PostCard.js [hunk 1]
```diff
@@ -5,57 +5,69 @@ import {
...
       <Box
         h='100%'
         w={'full'}
-        bg={useColorModeValue('white', 'gray.900')}
+        bg={'white'}
         p={8}
-        borderBottom={'1px'}
-        borderBottomColor={'gray.200'}
+        // borderBottom={'1px'}
+        // borderBottomColor={'gray.200'}
```
Comment (horsemaker) [inline src/frontend/components/post/PostCard.js:34]: Remove comments if not necessary.

File: src/frontend/features/user/userSlice.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import axios from "axios" ;
+import { createSlice , createAsyncThunk } from "@reduxjs/toolkit";
+
+const initialState = {
+	allUsers:[],
+    singleUser:null,
+    userPosts:[],
+    userStatus:'idle',
+	error: null,
```
Comment (horsemaker) [inline src/frontend/features/user/userSlice.js:9]: is this code indented properly?

File: src/frontend/utlis/timeFormatter.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      return Math.floor(interval) + " hours";
+    }
+    interval = seconds / 60;
+    if (interval > 1) {
+      return Math.floor(interval) + " minutes";
+    }
+    return Math.floor(seconds) + " seconds";
+  }
+
+export { timeFormatter  } ;
```
Comment (horsemaker) [inline src/frontend/utlis/timeFormatter.js:29]: Damn! Nice implementation. 👌

**Review by horsemaker — APPROVED** [review]
The UI looks smooth and clean. Great work here Narhari. 👍

File: src/frontend/components/post/PostCard.js (hunk 1 above)
Comment (narharikale) [reply src/frontend/components/post/PostCard.js:34]: it required

File: src/frontend/features/user/userSlice.js (hunk 2 above)
Comment (narharikale) [reply src/frontend/features/user/userSlice.js:9]: need fix formatter

**Pull request merged**